etc/docker/node-init.cni.sh

```diff
@@ -43,7 +43,7 @@ ln -snf /sys/fs/bpf /bpffs
 ln -snf /var/mizar/build/bin /trn_bin
 ln -snf /var/mizar/build/xdp /trn_xdp
 ln -snf /var/mizar/etc/cni/10-mizarcni.conf /etc/cni/net.d/10-mizarcni.conf
-ln -snf /var/mizar/mizar/cni.py /opt/cni/bin/mizarcni
+ln -snf /var/mizar/build/bin/mizarcni /opt/cni/bin/mizarcni
```

Comment:
Can you please rebase to pick up latest commit? We need to use the C2C change in order for things to work for containerd. (PR https://github.com/CentaurusInfra/mizar/pull/524/files).

Also, this should be added to Dockerfile now that it is a static binary and should be installed to /opt/cni/bin

Reply:
It has been rebased. C2C change is respected.